pages/projects/index.js

```diff
@@ -30,8 +30,7 @@ export default function Page() {
   const { title: siteTitle } = data?.generalSettings;
   const primaryMenu = data?.headerMenuItems?.nodes ?? [];
   const footerMenu = data?.footerMenuItems?.nodes ?? [];
-  const projectList = data.projects.edges.map((el) => el.node);
-
+  const projectList = data.projects.nodes;
```

Comment:
@josephfusco thanks for the review. I agree that would be good to do, but since this is already tested as is and will be merged soon for deploy, I'd rather not touch anything. It's an easy change later when we touch this again. I'll ticket it separately.